pkg/sfu/downtrack.go

```diff
@@ -237,6 +237,7 @@ type DownTrack struct {
 	isClosed             atomic.Bool
 	connected            atomic.Bool
 	bindAndConnectedOnce atomic.Bool
+	writable             atomic.Bool
```

Comment:
Making one bool that combines `bound` and `connected` as `WriteRTP` was check two atomics and `atomic.Load` was poking its head in the profile.